<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="searchBtn">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
   <widget class="QComboBox" name="category">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>카테고리별 찾기</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>한식</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>양식</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>일식</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>중식</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>분식</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>치킨</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>피자</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="searchtext">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reduncheck_window">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>500</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>중복 검사</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_window">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>430</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>대화 데이터 추가</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>570</y>
      <width>369</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="prevBtn">
       <property name="text">
        <string>prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="min_page">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pageEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="max_page">
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextBtn">
       <property name="text">
        <string>next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="excel_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>791</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
 </widget>
 <resources>
  <include location="../../../Downloads/myres.qrc"/>
 </resources>
 <connections/>
</ui>
